src/engine/types/button.ts

```diff
@@ -63,6 +67,10 @@ export class Button extends Type<ButtonDefinition> {
     }
 
     private setRect(rect: number[] | reference) {
+        if (this.gfxStandard) {
+            this.unregisterInteractive(this.gfxStandard)
```

Comment:
I guess that its not possible to undo SETRECT right?

Reply:
That's right. Should have put a comment.